### DISCUSSION TITLE
好好好，这下我也成为一名绳匠了

### DISCUSSION THREAD

**Discussion by Jiangxii1218** [share121/inter-knot#3178] [其他]
github用得少，这玩意可以发动图不，可以的话期望能够考虑更新一波
![pexels-athena-1858409](https://github.com/user-attachments/assets/cd0644c4-41ae-4274-bbea-e702e88f6443)

**Comment by cpu644** [comment]
可以动图

**Comment by Jiangxii1218** [comment]
但是为啥我看其他的帖子都是静止图啊，是没人发还是我加载不出来：(

**Comment by fashiye** [comment]
https://github.com/share121/inter-knot/discussions/3173#new_comment_form 
这篇帖子用的就是动图<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
